src/runtime/attachment_manager/index.ts

```diff
@@ -8,6 +8,8 @@ export interface IAttachment {
   media: messages.Media
 }
 
+export type IAttachFunction = (attachment: IAttachment) => void
```

Comment:
We can now use this to DRY up the `onAttachment` property  / attachment manager constructor

Reply:
Yup done